### PR TITLE
Sharded Stats Data

### PR DESCRIPTION
**Pull request by QuantumlyTangled**
This PR adds a 'data/stats' endpoint which can provides stats about the client but also gets the sharded data if needed.

### CODE REVIEW

File: src/routes/stats.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+module.exports = class extends Route {
+
+	constructor(...args) {
+		super(...args, { route: 'data/stats' });
+	}
+
+	async get(request, response) {
+		const data = {
+			users: await toShard(this.client.users.size, this.client, true, true),
+			members: await toShard(this.client.guilds, this.client, true, true),
```
Comment (UnseenFaith) [inline src/routes/stats.js]: Needs to be `this.client.guilds.members`

File: src/routes/stats.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+			guilds: await toShard(this.client.guilds.size, this.client, true, true),
+			channels: await toShard(this.client.guilds, this.client, true, true),
```
Comment (UnseenFaith) [inline src/routes/stats.js]: Needs to be `this.client.channels`

File: src/routes/stats.js (hunk 1 above)
Comment (UnseenFaith) [reply src/routes/stats.js]: Actually, needs to loop through all guilds and add the memberCount.

**Comment by QuantumlyTangled** [comment]
Ok, it worked in Quark yet this is diffrent so on that note it is getting closed.

**Pull request closed without merging**